suites/reef/rbd/tier-2_rbd_cache_scenarios.yaml

```diff
@@ -108,3 +108,20 @@ tests:
       module: test_rbd_immutable_cache.py
       name: Immutable object cache creation
       polarion-id: CEPH-83574134
+
+  - test:
+      abort-on-fail: true
+      config:
+        rep_pool_config:
+          num_pools: 2
+          num_images: 5
```

Comment:
default size of images created is 1GB is that enough for scale testing?

Reply:
Added `size to 4G`